src/web/config/migrations/008_create_user_email.rb

```diff
@@ -5,6 +5,7 @@
     String :email, :null => false
     index :email, :unique => true
     Boolean :verified, :null => true
+    String :verification_code, unique: true
```

Comment:
Could you please add a new migration file? Any change on db schema without new migration won't take effect on existing db